frontend/src/lib/widgetTheme.ts

```diff
@@ -239,6 +239,14 @@ export const buttonOverrides = {
   },
 }
 
+export const multiSelectOverrides = {
+  ValueContainer: {
+    style: () => ({
+      height: "44px", // Fixing height in order to prevent some changing height
```

Comment:
Is this due to a BaseWeb bug or something like that?

Reply:
Yes, it is a problem from BaseWeb